docs/application/tizen-studio/setup/hardware-accelerated-execution-manager.md

```diff
@@ -15,39 +15,30 @@ Hardware requirements:
 <a name="on_Windows"></a>
 ## Installing Intel&reg; HAXM on Windows&reg;
 
-Supported Windows&reg; versions: 8.1/8/7 (32/64-bit), Vista (32/64-bit), XP SP2 or higher (32-bit only)
+Supported Windows&reg; versions: 10/8.1/8/7 (32/64-bit), Vista (32/64-bit), XP SP2 or higher (32-bit only)
 
-**HAXM is installed automatically as part of the Tizen Studio**. If you want to install the HAXM individually, visit [http://download.tizen.org/sdk/haxm/6.0.4/win/](http://download.tizen.org/sdk/haxm/6.0.4/win/) and download the Windows installer package.
```

Comment:
**Rephrase** HAXM is installed automatically **to** HAXM is automatically installed 

**Rephrase** If you want to install the HAXM individually **to** If you want to install the HAXM separately